### ISSUE TITLE
IDEA for future contents and scaling

### ISSUE THREAD

**Issue by warius1983** [datchannin/Brothers-WoW#39]
This is just an idea that you might like. it's a little over powered at first glance but after some reasoning makes a lot of sense,

from the player.nut these are the experience points needed to level Up:

liv 1	0,
liv 2	200,
liv 3	500,
liv 4	1000,
liv 5	2000,
liv 6	3500,
liv 7	5000,
liv 8	7000,
liv 9	9000,
liv 10 12000,
liv 11 15000

Ok now I don't know the exact mathematichal series but for example in the end of liv 15 we will have something like this

liv 15 30000

This mean that your character has reached the veteran level, now a character should no more level up or if he levels up it could be almost useless, I tried during the first patches to set different amount of experience for levels over 15.

Now I'D say I continue the level UP like this,

liv 15 30000
liv 16 330000
liv 17 630000
liv 18 930000
liv 19 1230000

You may say what the hell how a player should be supposed to do 300k of experience to get a level and why?
This is my Idea, 14 perks are generally a good amaount to have a nice build for any background, but some of the times you make a WARRIOR TANK, WARRIOR DPS, MAGE FROST DPS and so on.

Yesterday you made to me a nice question, Why Should I do Gnolls? what is forcing by doing them if I have token obtainable in the open world? here is my new concept, there is no more a veteran level cap, every level you get a perk and stats, BUT how do I do 300k experience all of sudden?

Easy answer killing a gnoll camp grants a super potion of experience that a character can drink and step from level 15 to 16, 16 to 17 and so on but only one potion per battle, and if in the battle some brother die you can istantly level up a new brother to gap the loss one, of course only one at a time, so it's risky but may hep in making better future builds for harder contents without over powering too much the group, do you think it is something that people will like? this way you have a reason to fight gnolls!

**Comment by datchannin** [comment]
May it would be better to add some potion with fixed number of experience instead of +1 level (for example 7000 exp).
So player can use it for give lot exp number to "old" brother, that gives big step to have next level.
Or player can use it for "new" brother and give him fast boost +6 levels.

**Comment by warius1983** [comment]
Yes number need to be tuned, but that's why I put a fixed number of XP to level up after liv 15, one potion one level, one potion and you get a borther back from deat during encounter, this means that if you deceide to go after gnolls, you get some extra stuff. i will check experience level by level and give better numbers
